apps/mull-ui/src/constants.ts

```diff
@@ -4,14 +4,16 @@ import { EventRestriction, IEvent } from '@mull/types';
  * enum of the current routes for the application
  */
 export const ROUTES = {
-  HOME: '/home',
   MAP: '/map',
   CREATE_EVENT: '/create-event',
   TOOLS: '/tools',
   MESSAGES: '/messages',
   PROFILE: '/profile',
   LOGIN: '/login',
   REGISTER: '/register',
+  DISCOVER: '/discover',
+  UPCOMING: '/upcoming',
+  MYEVENTS: '/myevents',
```

Comment:
This is really minor but I think it should be consistent
```suggestion
  MY_EVENTS: '/my-events',
```